src/components/Sections/Home/Ourvoice/Container.js

```diff
@@ -0,0 +1,13 @@
+import ContainerLeft from './ContainerLeft';
+import ContainerRight from './ContainerRight';
+
+function Container() {
+  return (
+    <div className="ourvoice__container">
+      <ContainerLeft />
```

Comment:
This need to be just 1 component and pass properties to complete the data

<OurVoiceCard title="" etc etc etc/>